test/e2e/storage/external/README.md

```diff
@@ -32,18 +32,14 @@ prefix.
 To run just those tests for the example above, put that content into
 `/tmp/hostpath-testdriver.yaml` and invoke:
 
-    ginkgo -p -focus='External.Storage.*csi-hostpath' \
-           -skip='\[Feature:|\[Disruptive\]' \
-           ./test/e2e \
-           -- \
-           -storage.testdriver=/tmp/hostpath-testdriver.yaml
+    e2e.test -ginkgo.focus='External.Storage.*csi-hostpath' \
```

Comment:
One downside is that we lose the parallelism.  When I spoke with test-infra, they said that ginkgo -p flag will invoke the e2e binary multiple times with different -ginkgo.prallel.node values to manage the parallelism.

So I think we can just replace "./test/e2e" with the e2e binary in the original command

Reply:
I see, done

Reply:
My original intent was to keep this README.md generic, i.e. not assume that this package is used only in the Kubernetes e2e.test binary. But I guess anyone who does that is able to figure out how to generate an `e2e.test` binary or invoke `ginkgo` on the source, so this change is fine.